packages/cli/odahuflow/cli/utils/error_handler.py

```diff
@@ -0,0 +1,41 @@
+#
+#    Copyright 2020 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+import logging
+from contextlib import contextmanager
+import sys
+
+import click
+
+LOG = logging.getLogger(__name__)
+
+
+@contextmanager
+def cli_error_handler():
+    """
+    Print error message to stdout and exit if any error occurs
+    """
+    try:
+        yield
+    except Exception as e:
+        # If the verbose flag is enabled than the stacktrace will be logged.
+        LOG.exception('Exception occurs during CLI invocation')
+        # This message always appears in stdout.
+        # The verbose flag does not affect this.
+        click.echo(f'Error: {str(e)}')
```

Comment:
Let's add something like: "For more information rerun command with --verbose flag" ?

Reply:
Done.
The message must appear only when the flag is disabled. But I can determine the flag only by log level. It's a bit awful.